### ISSUE TITLE
IllegalStateError: Backoff in progress.

### ISSUE THREAD

**Issue by darrylhodgins** [grover/homebridge-dacp#4]
I'm attempting to use `homebridge-dacp` with an Apple TV 4K.  Setting up the pairing works, but then I get this…

Any ideas?

Thanks!

```
[2017-12-12 20:55:30] [DACP] The accessory Homebridge Apple TV Remote is announced.
[2017-12-12 20:55:30] [DACP] Attempting to reconnect to Homebridge Apple TV Remote in 0.1 seconds.
[2017-12-12 20:55:30] [DACP] The accessory Homebridge Apple TV Remote is announced.

/opt/node/lib/node_modules/homebridge-dacp/node_modules/precond/lib/checks.js:15
    throw error;
    ^
IllegalStateError: Backoff in progress.
    at Backoff.backoff (/opt/node/lib/node_modules/homebridge-dacp/node_modules/backoff/lib/backoff.js:38:13)
    at DacpAccessory.accessoryUp (/opt/node/lib/node_modules/homebridge-dacp/src/DacpAccessory.js:131:19)
    at _accessories.forEach.accessory (/opt/node/lib/node_modules/homebridge-dacp/src/index.js:68:19)
    at Array.forEach (<anonymous>)
    at DacpPlatform._onServiceUp (/opt/node/lib/node_modules/homebridge-dacp/src/index.js:66:23)
    at emitOne (events.js:116:13)
    at DacpBrowser.emit (events.js:211:7)
    at Browser._browser.on.service (/opt/node/lib/node_modules/homebridge-dacp/src/dacp/DacpBrowser.js:22:12)
    at emitTwo (events.js:126:13)
    at Browser.emit (events.js:214:7)
```

**Comment by grover** [comment]
Thanks for the bug report. I'll fix this in 0.7.4.

**Comment by grover** [comment]
I just published 0.7.4, which should hopefully take care of this issue. Feedback would be appreciated.

**Comment by darrylhodgins** [comment]
Works great.  Thanks, @grover!  And thanks for #5 too!
